src/views/Game/GameInfoModal.tsx

```diff
@@ -156,18 +156,7 @@ export class GameInfoModal extends Modal<Events, GameInfoModalProperties, {}> {
 
     rengoTeamList = () => (
         <div className="rengo-teams-container">
-            <div className="rengo-team-list">
-                <span className="team-title">Black Team</span>
-                {this.props.config.rengo_teams.black.map((player) => (
-                    <Player disableCacheUpdate icon rank user={player} />
-                ))}
-            </div>
-            <div className="rengo-team-list">
-                <span className="team-title">White Team</span>
-                {this.props.config.rengo_teams.white.map((player) => (
-                    <Player disableCacheUpdate icon rank user={player} />
-                ))}
-            </div>
+            {/* this space intentionally left blank: it doesn't work out to try to show rengo teams in this small modal */}
```

Comment:
Why not just delete the whole function?  Getting rid of that would simplify the caller a bit too:

```diff
- {this.props.config.rengo ? this.rengoTeamList() : this.twoPlayerTeamList()}
+ {this.props.config.rengo || this.twoPlayerTeamList()}
```